src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareApdexAnalysisIT.kt

```diff
@@ -0,0 +1,269 @@
+package com.atlassian.performance.tools.hardware
+
+import com.amazonaws.services.ec2.model.InstanceType
+import com.amazonaws.services.ec2.model.InstanceType.*
+import com.amazonaws.services.s3.transfer.TransferManagerBuilder
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.aws
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.logContext
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.taskName
+import com.atlassian.performance.tools.hardware.IntegrationTestRuntime.workspace
+import com.atlassian.performance.tools.hardware.failure.BugAwareTolerance
+import com.atlassian.performance.tools.jiraactions.api.*
+import com.atlassian.performance.tools.jirasoftwareactions.api.actions.ViewBacklogAction
+import com.atlassian.performance.tools.jvmtasks.api.TaskTimer.time
+import com.atlassian.performance.tools.lib.Apdex
+import com.atlassian.performance.tools.lib.table.GenericPlainTextTable
+import com.atlassian.performance.tools.lib.WeightedActionMetric
+import com.atlassian.performance.tools.lib.readResult
+import com.atlassian.performance.tools.lib.s3cache.S3Cache
+import com.atlassian.performance.tools.report.api.StandardTimeline
+import com.atlassian.performance.tools.report.api.result.EdibleResult
+import com.atlassian.performance.tools.report.api.result.RawCohortResult
+import com.atlassian.performance.tools.workspace.api.TestWorkspace
+import org.apache.logging.log4j.Logger
+import org.junit.Test
+import java.io.File
+import java.time.Duration
+import java.nio.file.FileSystems
+import java.nio.file.Paths
+
+
+class HardwareApdexAnalysisIT {
+
+    private val logger: Logger = logContext.getLogger(this::class.java.canonicalName)
+
+    // define the appdex sets we want to investigate
+    private val jiraInstanceTypeOfInterest: InstanceType = C518xlarge
+    private val nodeCountOfInterest: Int = 7
```

Comment:
We can define the entire `hardwareOfInterest: Hardware`
It also contains `db: InstanceType`
Did we skip DB filtering by design or by accident?

Reply:
By Accident I think, I will add that in.

Reply:
done 👍